packages/ui/src/views/repo/pull-request/details/components/conversation/sections/pull-request-changes-section.tsx

```diff
@@ -9,10 +11,65 @@ import {
 } from '@/views'
 import { cn } from '@utils/cn'
 import { getInitials } from '@utils/stringUtils'
+import { PanelAccordionShowButton } from '@views/repo/pull-request/details/components/conversation/sections/panel-accordion-show-button'
 import { isEmpty } from 'lodash-es'
 
 import { LineDescription, LineTitle } from './pull-request-line-title'
 
+const getStatusIcon = (status: string) => {
+  switch (status) {
+    case 'pending':
+      return <Icon name="pending-clock" className="text-icons-alert" />
+    case 'warning':
+      return <Icon name="triangle-warning" className="text-tertiary-background" />
+    case 'error':
+      return <Icon name="triangle-warning" className="text-destructive" />
+    default:
+      return <Icon name="success" className="text-foreground-success" />
+  }
+}
+
+interface HeaderItemProps {
+  header: string
+}
+
+const HeaderItem: FC<HeaderItemProps> = ({ header }: HeaderItemProps) => {
+  return <span className="text-12 text-foreground-1">{header}</span>
+}
+
+interface AvatarItemProps {
+  evaluations: TypesOwnerEvaluation[] | undefined
+}
+
+const AvatarItem: FC<AvatarItemProps> = ({ evaluations }: AvatarItemProps) => {
+  return (
+    <StackedList.Field
+      className="pb-0"
+      title={
+        <div className="flex items-center">
+          {evaluations &&
+            evaluations.map(({ owner }, idx) => {
+              if (idx < 2) {
+                return (
+                  <Avatar.Root key={owner?.id || idx}>
+                    <Avatar.Fallback>{getInitials(owner?.display_name || '')}</Avatar.Fallback>
+                  </Avatar.Root>
+                )
+              }
+              if (idx === 2 && evaluations.length && evaluations.length > 2) {
+                // TODO: do popover with all the names
+                return <span key={owner?.id} className="text-12">{`+${evaluations.length - 2}`}</span>
```

Comment:
Template string seems unnecessary here.
```suggestion
                return <span key={owner?.id} className="text-12">+{evaluations.length - 2}</span>
```